pageserver/src/tenant.rs

```diff
@@ -1316,7 +1328,19 @@ impl Tenant {
         ctx: &RequestContext,
     ) -> anyhow::Result<Arc<Timeline>> {
         let uninit_tl = self.create_empty_timeline(new_timeline_id, initdb_lsn, pg_version, ctx)?;
+
+        // Setup minimum keys required for the timeline to be usable.
+        let mut modification = uninit_tl
+            .raw_timeline()
+            .expect("we just created it")
+            .begin_modification(initdb_lsn);
+        modification.init_empty().context("init_empty")?;
+        modification
+            .commit()
+            .context("commit init_empty modification")?;
+
```

Comment:
In `pgdatadir_mapping::create_test_timeline`, there's also a call to `init_empty` which is now redundant with this.

Reply:
Yep, learned that, fixing it.